server/controllers/email-controller.js

```diff
@@ -158,7 +158,7 @@ exports.getListOfLectures = async (from, to) => {
     .andWhere("lecture.start", "<", to)
     .groupBy(
       "lecture.id",
-      "lecture.name",
+      //"lecture.name",
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.